data_structures/binarysearchtree/binarysearchtree.go

```diff
@@ -47,66 +48,66 @@ func (b *BinarySearchTree) Insert(data int) *BinarySearchTree {
 func (b *BinarySearchTree) BFSTraversal() []int {
 	visited := []int{}
 
-	if b.root == nil {
+	if b.Root == nil {
```

Comment:
Looks good to me... but please make all the related changes in files depending on this module in the next update.